cypress/integration/videoRegionSpec.js

```diff
@@ -0,0 +1,30 @@
+import { getBlockByType } from '../support/bodyTestHelper';
+
+const getBlockData = (blockType, win) => {
+  const { blocks } = win.SIMORGH_DATA.pageData.content.model;
+
+  return getBlockByType(blocks, blockType);
+};
+
+describe('Video', () => {
```

Comment:
Will fail on live, needs to be limited to test and local

Reply:
yeah, @benjaminhobbs it will surely fail! this is just a draft proof for playing video with cypress